README.md

```diff
@@ -36,3 +36,45 @@ npm run build
 You can preview the production build with `npm run preview`.
 
 > To deploy your app, you may need to install an [adapter](https://kit.svelte.dev/docs/adapters) for your target environment.
+
+---
+
+# Actual README Content
+
+## Ubiquitous Language
+- Campaign: A story co-written by a game master and their players about the non-player characters the GM runs an the player characters the players run.  Could be based in any TTRGP series or home brew game.
+- Session: A period of continuous time when the GM and players got together to advance the story/game.
+- Game master: the user who creates a campaign and owns the right to dole out or restrict editing permissions.
+- Player: other users who may view the campaign.  Depending on their access permissions, they may also be able to edit quests and milestones.
+- Quest: A major goal one or more players are pursuing.  Can be active or completed.
+	- Current Goal: a minor goal one or more players is pursing to achieve a major goal.  Current goals may shift or change without being achieved.  
+- Milestone: actions that the player characters took in the service of one or more goals (not always intentionally)
```

Comment:
Ah, I see now.  Maybe you made some grand gesture that affected several of your goals.

I have trouble picturing an elegant UI / flow for associating them... but that can be a problem for later.

Reply:
Do you think that, when I click a quest, it'll look like this?

[ Quest title and description ]

[Goals in order from oldest to newest]

[Divider]

[Milestones in order from newest to oldest]

...I'm picturing the milestones as being similar to journal entries, whereas goals are more like todos.

Reply:
If you can convince me that historical goals are worth maintaining, then I would interlace them with their milestones.  Something like:

[ Quest title and description ]
[first goal]
- [milestone]
- [milestone]
- [milestone]
[first goal]
- [milestone]
- [milestone]
[current goal 1]
- [milestone]
[current goal 2]
- [milestone]

Reply:
> I have trouble picturing an elegant UI / flow for associating them... but that can be a problem for later.

We can always reswizzle the data into client-side objects.  We don't have to use is as is.   It may be best to have the UI only talk to quests in some cases and let quests be able to return data for the milestones they're associated with as needed.

Reply:
I'm not convinced that it's necessary to remember historical goals.  After considering your example with the deceptive quest giver, I see the usefulness of goals being fully malleable and able to bend to the whims of the story.

I still feel like there might be some satisfaction inherent in a "complete goal" button of some kind.

For the UI display...  
We can keep thinking on it™